api/src/controllers/roles.ts

```diff
@@ -73,6 +73,36 @@ const readHandler = asyncHandler(async (req, res, next) => {
 router.get('/', validateBatch('read'), readHandler, respond);
 router.search('/', validateBatch('read'), readHandler, respond);
 
+router.get(
+	'/me',
+	asyncHandler(async (req, res, next) => {
+		if (!req.accountability?.user && !req.accountability?.role) throw new ForbiddenError();
+
+		const service = new RolesService({
+			accountability: req.accountability,
+			schema: req.schema,
+		});
+
+		const query = { ...req.sanitizedQuery, limit: -1 };
+
+		try {
+			const roles = await service.readMany(req.accountability.roles, query);
```

Comment:
Should we include this comment about it being temporary to be consistent with other usages?

```suggestion
		// TODO fix this at the service level
		const temporaryQuery = { ...req.sanitizedQuery, limit: -1 };

		try {
			const roles = await service.readMany(req.accountability.roles, temporaryQuery);
```